packages/ra-core/src/controller/input/referenceDataStatus.ts

```diff
@@ -21,19 +21,21 @@ export const getStatusForInput = ({
     referenceRecord,
     translate = x => x,
 }: GetStatusForInputParams) => {
+    // @rattrayalex several good examples in this file.
+    // @rattrayalex: don't love having this function call break here, should bump.
     const matchingReferencesError = isMatchingReferencesError(
         matchingReferences
-    )
-        ? translate(matchingReferences.error, {
-              _: matchingReferences.error,
-          })
-        : null;
+    ) ?
```

Comment:
I think ideally if a function call would break here, it should bump to the next line…